app/views/users/show.html.erb

```diff
@@ -1,3 +1,31 @@
-<h1>User</h1>
 
-<p>Here is a given user</p>
\ No newline at end of file
+<section>
+<div class="user" >
+    <img class="pro-pic" src='<%@user.photo%>' alt='Profile-pic'> <br>
+  <div class="user-dets">
+    <h2 class="user-name"><%=@user.name%></h2>
+    <p class='num-posts' >Number of Posts: <%=@user.posts_counter%></p>
+  </div>
+</div>
+
+<div class="user-bio">
+<%= @user.bio %> <br>
+</div>
+
+<br>
+
+<% @user.three_most_recent_posts.each do |post| %>
+  <div class="user-post">
+    <h4><%=post.title%></h4>
+    <div>
+      <p><%=post.text%></p>
+      <p>Comments:<%=post.comments_counter%></p>
+      <p>Likes:<%=post.likes_counter%></p>
+    </div>
+  </div>
+  <br>
+<%end%>
+
+
+<button>See all posts</button>
```

Comment:
- [ ] Also the `button`, It can be good to make it navigate to All pasts, it will be something like this:

````ruby
<%= link_to "See All Posts", user_posts_path(@user) %>
````

You may replace the line `30` with the code above ⬆️

Reply:
Well resolved ✔️